flash_all.sh

```diff
@@ -4,13 +4,18 @@ set -e
 DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
 cd $DIR
 
+EDL_DIR=$DIR/edl
+if [ ! -d  $EDL_DIR ]; then
+  ./install_edl.sh
+fi
+
 for part in aop xbl xbl_config devcfg; do
-  fastboot flash ${part}_a $DIR/agnos-firmware/$part.bin
-  fastboot flash ${part}_b $DIR/agnos-firmware/$part.bin
+  $EDL_DIR/edl w ${part}_a $DIR/agnos-firmware/$part.bin
+  $EDL_DIR/edl w ${part}_b $DIR/agnos-firmware/$part.bin
 done
 
 ./flash_bootloader.sh
 ./flash_kernel.sh
 ./flash_system.sh
 
-fastboot continue
+$EDL_DIR/edl reset
```

Comment:
i'd make the env var directly point to `edl` like this
```suggestion
$EDL reset
```